modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/internal/blueprint/search/spi/searcher/SXPBlueprintSearchRequestContributor.java

```diff
@@ -69,16 +69,16 @@ private void _contributeSXPBlueprintExternalReferenceCode(
 					object);
 		}
 
-		if (object == null) {
-		}
-		else if (object instanceof String) {
+		if (object instanceof String) {
 			String string = (String)object;
 
 			if (Validator.isNotNull(string)) {
 				_enhanceWithExternalReferenceCode(
 					searchRequestBuilder, StringUtil.split(string));
 			}
 		}
+		else if (object == null) {
```

Comment:
This part (what I was previously meaning) could be simplified to:

```
		if (object instanceof String) {
			String string = (String)object;

			if (!Validator.isBlank(string)) {
				_enhanceWithExternalReferenceCode(
					searchRequestBuilder, StringUtil.split(string));
			}
		}
		else if (object != null) {
			throw new IllegalArgumentException(
				"Invalid search experiences blueprint External Reference " +
					"Code " + object);
		}
```

And the same in the old code, `_contributeSXPBlueprintId () `
```
		if (object instanceof Number) {
			_enhance(searchRequestBuilder, GetterUtil.getLong(object));
		}
		else if (object instanceof String) {
			String string = (String)object;

			if (Validator.isNotNull(string)) {
				_enhance(
					searchRequestBuilder,
					GetterUtil.getLongValues(StringUtil.split(string)));
			}
		}
		else if (object != null) {
			throw new IllegalArgumentException(
				"Invalid search experiences blueprint ID " + object);
		}
```

Those void `if`s are just stealing space.